utils/bots/TutorBot/cogs/TutorStaff.py

```diff
@@ -28,12 +28,73 @@ def __init__(self, bot):
         self.bot = bot
         self.est = pytz.timezone("US/Eastern")
 
+    @slash_command(
+        name="schedule",
+        description="Create a Tutor Session",
+        guild_ids=[TUT_ID.g_tut],
+    )  # SLASH CMD FOR TUTOR SERVER
+    async def _schedule__(
+            self,
+            ctx,
+            date: Option(str, "Enter a date in MM/DD format. EX: 02/02"),
+            time: Option(str, "Enter a time in HH:MM format. EX: 3:00"),
+            ampm: Option(str, "AM or PM", choices=["AM", "PM"]),
+            student: Option(int, "Enter the student you'll be tutoring for this session."),
```

Comment:
Replace param description to `"Enter the student ID you'll be tutoring for this session."`

Reply:
fixed